### PR TITLE
Fix utils.center_mass_exp and added a test code

### PR DESCRIPTION
**Pull request by lcnature**

### CODE REVIEW

File: brainiak/utils/utils.py [hunk 1]
```diff
@@ -627,6 +631,9 @@ def center_mass_exp(a, b, scale=1.0):
         distribution.
     """
     assert a < b, 'a must be smaller than b'
+    assert b > 0, 'b must be larger than 0'
+    assert scale > 0, 'scale must be positive'
+    assert a >= 0, 'a cannot be smaller than 0'
```
Comment (mihaic) [inline brainiak/utils/utils.py]: These restrictions should be documented in the parameters section of the function docstring.

**Comment by mihaic** [comment]
@lcnature, @QihongL, will you implement Javier's suggestion? If not, I will merge this PR as is.

**Comment by QihongL** [comment]
I will try that today! Seems to be a good one to get familiarized with the contribution process!

File: brainiak/utils/utils.py (hunk 1 above)
Comment (QihongL) [reply brainiak/utils/utils.py]: By the way, regarding this section: if a >= 0, and b > a, then b must  > 0, so we don't need to check b > 0. That said, I actually think the current setting return more informative error messages, so I actually like it. Is this what we want?

Comment (mihaic) [reply brainiak/utils/utils.py]: Let's skip the unnecessary check.

**Comment by mihaic** [comment]
Superseded by PR #288. Thanks for the prompt replies, @lcnature.

**Pull request closed without merging**